rafs/src/builder/core/v5.rs

```diff
@@ -187,7 +173,7 @@ impl Bootstrap {
 
         // Set super block
         let mut super_block = RafsV5SuperBlock::new();
-        let inodes_count = bootstrap_ctx.inode_map.len() as u64;
+        let inodes_count = bootstrap_ctx.inode_map.len() as u64 + 1;
```

Comment:
Can you separate this commit into a different PR, and add a full analysis of it? It is quite easy to get slipped in w/o notice but we definitely need to figure out what happened and get it backported.

Reply:
A dedicated PR for it: https://github.com/dragonflyoss/image-service/pull/1275